packages/tex-to-myst/src/frontmatter.ts

```diff
@@ -281,6 +282,17 @@ const FRONTMATTER_HANDLERS: Record<string, Handler> = {
       .map((k) => k.trim())
       .filter((k) => !!k);
   },
+  macro_newtheorem(node, state) {
+    // https://tex.stackexchange.com/questions/155710/understanding-the-arguments-in-newtheorem-e-g-newtheoremtheoremtheoremsec/155714#155714
+    const [, nameNode, x, labelNode, y] = node.args ?? [];
+    const name = texToText(nameNode);
+    const label = texToText(labelNode);
+    const countWith = texToText(x) || undefined;
+    const countAfter = texToText(y) || undefined;
+    state.data.theorems[name] = { label, countWith, countAfter };
```

Comment:
I feel like I'm missing where these theorems stored on `data` are actually consumed... 🤔 Maybe they just aren't yet...?

Reply:
They are not yet, correct. When we start enumerating theorems in a custom way, they this count information will likely be used.